packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,29 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCount: 0 }),
+  effects: {
+    async forceRefreshXoaStatus() {
+      await clearXoaCheckCache()
+      await this.effects.refreshXoaStatus()
+    },
+    refreshXoaStatus() {
+      this.state.checkXoaCount += 1
+    },
+  },
+  computed: {
+    xoaStatus: {
+      get({ checkXoaCount }) {
+        // To avoid aggressive minification which would remove destructuration
+        noop(checkXoaCount)
+        return getXoaPlan() === 'Community' ? '' : checkXoa()
```

Comment:
The plugin may not be loaded so an error may be thrown here.

Reply:
https://github.com/vatesfr/xen-orchestra/blob/9226c6cac193f9ed9dd9a4fd1e2c7a9833e7f6c6/packages/xo-web/src/common/notification.js#L11

Display `error` notification to the user could be relevant ?